PIDs:
  X:
    kp: 0.42
    kd: 0.05
    ki: 0.2
    kpp: 3.3
    ff: 0.6
    minOutput: -10.0
    maxOutput: 10.0
    integratorMin: -1000.0
    integratorMax: 1000.0
  Y:
    kp: 0.42
    kd: 0.05
    ki: 0.2
    kpp: 3.3
    ff: 0.6
    minOutput: -10.0
    maxOutput: 10.0
    integratorMin: -1000.0
    integratorMax: 1000.0
  Z:
    kp: 2.2
    kd: 0.0
    ki: 0.1
    kpp: 2.38
    ff: 0.5
    minOutput: -10.0
    maxOutput: 10.0
    integratorMin: -1000.0
    integratorMax: 1000.0

